docs/toc/processor.rst

```diff
@@ -0,0 +1,95 @@
+Processor
+==========
+
+A pipeline component that wraps inference model and set up inference related work.
+
+
+
+Usage
+------
+
+* wrap model by setting `self.model`
+* set up inference related behaviors
+
+
+Functions
+------------------
+
+* `initialize()`: initilize model and inference related attributes.
+* `process()`: using loaded model to make predictions and write the prediction results out
+
+
+
+
+Processor Class Hierarchy
+------------------------------
+
+Here we provide a simplified class hierarchy for :class:`PlainTextReader` to show the relations of processors which are subclasses of `PipelineComponent`.
```

Comment:
Why showing a hierarchy? And this is also not a hierarchy for readers

Reply:
It's supposed to be `MachineTranslationProcessor`. The hierarchy is to show it's essentially a `PipelineComponent`, and how to decide what processor to inherit when users need to customize their own.

Reply:
This should probably be explained better, a class hierarchy would be too boring for a reader